addons/oci-private-dns/README.md

```diff
@@ -0,0 +1,160 @@
+
+&nbsp;
+
+## **1. Single-Region: Private DNS configuration view**
+&nbsp;
+Configuration details:
+  - Hub VCN consists of the following resources and components: 
+    - Forwarding (**hub_dns_forwarder**) and Listening (**hub_dns_listener**) endpoints.
+    - Hub VCN Resolver has Associated private views for Hub and Spokes VCNs. As a result, it contains all DNS data/records for the three VCNs and can resolve any FQDN within the Hub and Spoke architecture.
+  - Spoke VCN Resolvers include
+    - Forwarding endpoints (**p_dns_forwarder** and **pp_dns_forwarder** respectively).
+    - Conditional forwarding rules, that directs queries for **oraclevcn.com**, **oraclecloud.com** and **oci.customer-oci.com** domains to the **hub_dns_listener** for resolution.
+
+<img src="images/one-region.png" width="900" height="value">
+
+&nbsp;
+
+#### DNS configuration with On-Premise connectivity:
+In addition to the above configuration, the following setup includes forwarding rules for On-Premises DNS zones in both - the Spoke VCNs and the Hub VCN. These rules direct queries to a Network Load Balancer (NLB), which serves as the target for forwarding. The On-Premises DNS servers are configured as backends for the DNS NLB.
+
+<img src="images/onprem.png" width="900" height="value">
+
+
+&nbsp;
+
+&nbsp;
+
+## Single-Region: Private DNS animation
+
+These animations illustrate the DNS query and response within Hub & Spoke, and covers the following scenarios:
+
+#### Scenario 1: DNS resolution within the same Spoke VCN
+- The **web01-p.ssnpweb.vcnprod.oraclevcn.com** instance in Prod Spoke VCN performs *nslookup* to retrieve an IP address of the **db01-p.ssnpdb.vcnprod.oraclevcn.com** database instance, which is located in the same Spoke VCN but in a different subnet.
+- Prod VCN resolver evaluates the query based on the [VCN DNS Resolver processing order](#VCN-DNS-Resolver-processing-order) as follows:<br>
+   ⓵ **Associated Private Views** - Since no Private view is associated with the Prod VCN resolver, it proceeds to the next step.<br>
+   ⓶ **Default Private View** - The default private view contains a DNS record for the database, so the resolver retrieves the record.
+- The Prod VCN resolver returns the DNS response to **web01-p**.
+
+<img src="https://github.com/oracle-quickstart/terraform-oci-open-lz/blob/content/addons/oci-private-dns/withinspoke.gif" width="800" />
+
+&nbsp;
+
+#### Scenario 2: Spoke to Spoke DNS resolution
+- The **web01-p.ssnpweb.vcnprod.oraclevcn.com** in prod Spoke VCN performs *nslookup* to retrieve the IP address of **web02-pp.ssnppweb.vcnpreprod.oraclevcn.com**, located in the preprod Spoke VCN.
+- Prod VCN resolver evaluates the query based on the [VCN DNS Resolver processing order](#VCN-DNS-Resolver-processing-order) as follows:<br>
+   ⓵ **Associated Private Views** - Since no Private view is associated with the Prod VCN resolver, it moves to the next option.<br>
```

Comment:
This looks a bit weird on my PC. why just not use 1. 2. I think it should be possible to have ordered list inside unordered list

https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#nested-lists